db/migrate/20160911210500_change_courses_end_to_datetime.rb

```diff
@@ -0,0 +1,12 @@
+class ChangeCoursesEndToDatetime < ActiveRecord::Migration
+  def up
+    change_column :courses, :end, :datetime
+    Course.all.each do |course|
```

Comment:
Let's do this data migration in a separate migration. That will reduce the risk of breaking the migrations if travis times out during the migration.